docs/API/InstrumentFormat.md

```diff
@@ -287,32 +342,188 @@ A NumericElement represents a numeric data input and has the general form of:
 
 
 
-### 2.1.5: ScoreFieldElement
+### 2.1.5: CalcFieldElement
 
-A score field represents a placeholder to display/save values based on other
+A calc field represents a placeholder to display/save values based on other
```

Comment:
There's no reason for this change. They're called "score fields" everywhere else in LORIS.